[2021-10-16 13:05:09,346] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T11:10:00+00:00 [queued]>
[2021-10-16 13:05:09,412] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T11:10:00+00:00 [queued]>
[2021-10-16 13:05:09,412] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:09,412] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:05:09,412] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:09,491] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_1> on 2021-10-16 11:10:00+00:00
[2021-10-16 13:05:09,501] {standard_task_runner.py:52} INFO - Started process 609 to run task
[2021-10-16 13:05:09,525] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_1', 'scheduled__2021-10-16T11:10:00+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmpjcctrx8q', '--error-file', '/tmp/tmpgd2gjsts']
[2021-10-16 13:05:09,525] {standard_task_runner.py:77} INFO - Job 327: Subtask get_gross_year_1
[2021-10-16 13:05:09,636] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T11:10:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:05:09,814] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T11:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T11:10:00+00:00
[2021-10-16 13:05:09,854] {python.py:152} INFO - Done. Returned value was: ['$54.53 ', '2010\n']
[2021-10-16 13:05:09,942] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_1, execution_date=20211016T111000, start_date=20211016T130509, end_date=20211016T130509
[2021-10-16 13:05:10,023] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:05:10,172] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
